### PR TITLE
Icon updates

### PR DESCRIPTION
**Pull request by VikChawla**
**Description**

_Write a short description of the changes included in this pull request._

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Progresses #???

### CODE REVIEW

File: app/components/UI/FiatOnRampAggregator/components/PaymentOption.tsx [hunk 1]
```diff
@@ -137,14 +152,26 @@ const PaymentOption: React.FC<Props> = ({
...
+              {id === '/payments/sepa-bank-transfer' && (
                 <Image source={sepa} style={styles.cardIcon} />
               )}
+              {(id === '/payments/ach-bank-transfer' ||
+                id === '/payments/bank-account') && (
+                <Image source={ach} style={styles.cardIcon} />
+              )}
+              {id === '/payments/upi' && (
+                <Image source={upi} style={styles.cardIcon} />
+              )}
```
Comment (wachunei) [inline app/components/UI/FiatOnRampAggregator/components/PaymentOption.tsx:174]: can you transform this into a component with a switch statement please? default cases must be `'/payments/apple-pay'` and `'/payments/debit-credit-card'`

**Review by wachunei — CHANGES_REQUESTED** [review]

File: app/components/UI/FiatOnRampAggregator/components/PaymentOption.tsx (hunk 1 above)
Comment (VikChawla) [reply app/components/UI/FiatOnRampAggregator/components/PaymentOption.tsx:174]: Okay got it

Comment (VikChawla) [reply app/components/UI/FiatOnRampAggregator/components/PaymentOption.tsx]: updated ✅

File: app/components/UI/FiatOnRampAggregator/components/PaymentTypeIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    }
+    case '/payments/upi': {
+      return <Image source={upi} style={style} />;
+    }
+    case '/payments/ach-bank-transfer':
+    case '/payments/bank-account': {
+      return <Image source={ach} style={style} />;
+    }
+    default:
+      return (
```
Comment (wachunei) [inline app/components/UI/FiatOnRampAggregator/components/PaymentTypeIcon.tsx:48]: Explicitly add these 
```suggestion
    }
    case '/payments/apple-pay':
    case '/payments/debit-credit-card':
    default:
      return (
```

**Review by wachunei — DISMISSED** [review]

**Pull request merged**